hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/net/NetUtils.java

```diff
@@ -98,7 +99,19 @@ public static SocketFactory getSocketFactory(Configuration conf,
     SocketFactory factory = null;
 
     String propValue =
-        conf.get("hadoop.rpc.socket.factory.class." + clazz.getSimpleName());
+        conf.get("hubspot.hadoop.rpc.socket.factory.class." + clazz.getSimpleName(),
+            conf.get("hadoop.rpc.socket.factory.class." + clazz.getSimpleName()));
```

Comment:
These two lines changed for interoperability with the cdh5 counterpart NetUtils. When cdh5 Hadoop libraries are unused, I can revert this.